addons/mail/static/src/discuss/voice_message/common/voice_analyser.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.VoiceAnalyser" owl="1">
+        <div class="d-flex align-items-center my-1 position-relative">
+            <t t-set="playOrPause"><t t-if="state.playing">Pause</t><t t-else="">Play</t></t>
+            <button class="border-0 btn rounded-pill fa ms-2 bg-100"
+                t-att-class="state.playing ? 'fa-stop text-danger btn-sm' : 'fa-play btn-light btn-sm'"
```

Comment:
Small resizing when start/stop.
This happens because `fa` icons do not have same width. Should use additional classname `fa-fw` to make font-awesome icons of same width, so there's no resizing.